sites/plasticsmachinerymagazine/config/site.js

```diff
@@ -0,0 +1,58 @@
+module.exports = {
+  logo: {
+    src: 'https://cdn.plasticsmachinerymagazine.com/files/base/ebm/pmm/image/static/logo/site_logo.png',
+    srcset: [
+      'https://cdn.plasticsmachinerymagazine.com/files/base/ebm/pmm/image/static/logo/site_logo.png 2x',
+    ],
+  },
+  primaryNavItems: [
+    { href: '/additive-manufacturing', label: 'Additive Manufacturing' },
+    { href: '/blow-molding', label: 'Blow Molding' },
+    { href: '/thermoforming', label: 'Thermoforming' },
+    { href: '/compounding', label: 'Compounding' },
+    { href: '/injection-molding', label: 'Injection Molding' },
+    { href: '/extrusion', label: 'Extrusion' },
+    { href: '/recycling', label: 'Recycling' },
+    { href: '/molds-tooling', label: 'Molds & Tooling' },
+    { href: '/rotomolding', label: 'Rotomolding' },
+  ],
+  secondaryNavItems: [
+    { href: '/subscribe', label: 'Subscribe' },
+    { href: '/magazine', label: 'In Print' },
+    { href: '/advertise', label: 'Advertise' },
+    { href: '/contact-us', label: 'Contact Us' },
+  ],
+  homeSections: [
+    { alias: 'additive-manufacturing', name: 'Additive Manufacturing' },
+    { alias: 'blow-molding', name: 'Blow Molding' },
+    { alias: 'thermoforming', name: 'Thermoforming' },
+    { alias: 'compounding', name: 'Compounding' },
+    { alias: 'injection-molding', name: 'Injection Molding' },
+    { alias: 'extrusion', name: 'Extrusion' },
+    { alias: 'recycling', name: 'Recycling' },
+    { alias: 'molds-tooling', name: 'Molds & Tooling' },
+    { alias: 'rotomolding', name: 'Rotomolding' },
+  ],
+  googleTagManagerID: 'GTM-KBM372Z',
```

Comment:
There's a lot of debate over whether `ID` within a camel-cased variable should be `ID` or `Id`. We tend to interchange it quite often. I'm okay with it if you are.

Reply:
Though, to be fair, we tend to use `Id` more than `ID`

Reply:
I think ID looks wrong, but...